### PR TITLE
[FIX] _find_manifest_path with directory less than 5 deep if no manifest found

### PR DESCRIPTION
**Pull request by petrus-v**
This happens if the test module is less than 5 directories deep from root.

### CODE REVIEW

**Comment by petrus-v** [comment]
hi @simahawk do you mind to activate the workflow in order to make sure test are running properly ?

I'm wondering why this project require maintainer approval to execute such test, do you worries about abuses or a missed configuration?

**Review by yvaucher — APPROVED** [review]

**Comment by yvaucher** [comment]
For those tests I don't think we need to import Odoo as we it's quite heavy. Thus I have added a mock of the imported odoo modules here #75

**Comment by yvaucher** [comment]
Please rebase

#75 should now fix the CI

**Comment by yvaucher** [comment]
Tested locally, I'll rebase and merge

**Pull request merged**
